tests/testthat/test_spcv_buffer.R

```diff
@@ -18,6 +18,8 @@ test_that("grouping throws errors when 'groups' and 'stratify' is set", {
 
 test_that("train and test set getter functions are working", {
   spcv_rsp_buffer = mlr_resamplings$get("spcv-buffer")
+  spcv_rsp_buffer$param_set$values = insert_named(spcv_rsp_buffer$param_set$values,
+    list(theRange = 1))
```

Comment:
does the user also need to use `insert_named()`?

Reply:
Only if the user does not want to overwrite his old settings of other parameters.